README.md

```diff
@@ -61,7 +81,29 @@ New Realm > Realm Settings > Login
 | Login with email  | true             |
 | Require SSL       | external request |
 
-New Realm > Realm Settings > Email
+[More about configuration](http://www.mastertheboss.com/keycloak/getting-started-with-keycloak-powered-by-quarkus/)
+
+**6. Symlink helsinki theme into theme folder**
+
+When you are in `<Keycloak folder>/themes`
+
+```bash
+ln -s ~/path/to/helsinki-keycloak-theme/helsinki
+```
```

Comment:
Did you get symlinking the theme work somehow? I remember trying it in many ways but never succeeded to make it work.

Reply:
I've been mostly developing via Docker and then files are copied with `docker cp`. I tried symlink directly with the Keycloak server few times (in 2020(?)). It did actually work.

Perhaps it would work with `docker compose` if symlink is in the right volume...

Reply:
So, do we want to keep this step 6 as it is? AFAIK it doesn't work.